test/functional/sqlsrv/MsHelper.inc

```diff
@@ -253,7 +253,7 @@ function getDefaultColname($dataType)
  */
 function getInsertSqlComplete($tbname, $inputs)
 {
-    $colStr = "INSERT INTO $tbname (";
+    $colStr = "INSERT INTO [$tbname] (";
```

Comment:
why need square brackets?

Reply:
One of the sqlsrv tests use a table name that begins with a number. Without square brackets that results in a sql error